fletcher/base.py

```diff
@@ -222,11 +221,31 @@ def __array__(self, dtype=None, copy=False):
         Returns
         -------
         np.ndarray
+
         """
         return np.array(
             self.data.to_pandas(deduplicate_objects=True), dtype=dtype, copy=copy
         )
 
+    def __arrow_array__(self, type=None):
+        # type: (pa.DataType,) -> pa.Array
+        """
+        implements pyarrow array interface (requires pyarrow>=0.15)
```

Comment:
```suggestion
        Implement pyarrow array interface (requires pyarrow>=0.15).
```
Flake8 is very rigorous about docstring of functions